dbus/methods_test.go

```diff
@@ -642,6 +651,49 @@ func TestEnableDisableUnit(t *testing.T) {
 	}
 }
 
+// Ensure that ListJobs works.
+func TestListJobs(t *testing.T) {
+	service := "oneshot.service"
+
+	conn := setupConn(t)
+
+	setupUnit(service, conn, t)
+	linkUnit(service, conn, t)
+
+	_, err := conn.StartUnit(service, "replace", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	jobs, err := conn.ListJobs()
+	if err != nil {
+		t.Skip(err)
+	}
+
+	found := isJobPresent(jobs, service)
+	if !found {
+		t.Fatalf("%s job not found in list", service)
+	}
```

Comment:
Can we also check the fields of the job we found here? Making sure that they aren't some dummy default value would be good already.

Reply:
Done